src/js/app.js

```diff
@@ -1,26 +1,31 @@
 import { buildData, filterObjectsOnNodeClick, filterNodes } from './data-builder';
 import renderNetwork from './network-vis';
 import renderHierarchy from './hierarchy-vis';
+import '../../node_modules/jquery-ui-dist/jquery-ui';
+import '../../node_modules/jquery-ui-dist/jquery-ui.min.css';
 
-let data = {};
+// Needed for node search functionality
+let allNodeNames = [];
 
+let data = {};
 const init = (rawData) => {
   data.rawData = rawData;
   data.hierarchyFiltered = false;
   data.nodesFiltered = false;
   data.formattedData = buildData(rawData);
   data.hierarchy = data.formattedData.hierarchy;
   data.renderedNetwork = renderNetwork(data.formattedData.nodes, data.formattedData.edges);
+  data.formattedData.nodes.map((x) => {
+    allNodeNames.push({ label: x.name, value: x.id });
+  });
   renderHierarchy(data.hierarchy);
 };
 const onNodeClicked = (currentNode, allChildren = []) => {
   if (currentNode.length > 0) {
-    if (!data.nodesFiltered) {
```

Comment:
add a comment explaining what you're doing here

Reply:
Done